plugins/vpc-shared-eni/network/bridge_windows.go

```diff
@@ -36,6 +37,18 @@ const (
 	hnsEndpointNameFormat = "cid-%s"
 )
 
+// NSType identifies the namespace type for the containers.
+type NSType int
+
+const (
+	// infraContainerNS identifies an Infra container NS for networking setup.
+	infraContainerNS NSType = iota
+	// nonInfraContainerNS identifies sharing of infra container NS for networking setup.
+	nonInfraContainerNS
+	// hcsNamespace identifies HCS NS for networking setup.
```

Comment:
`hcn` namespace?

Reply:
The namespaces are referred to as HCN namespaces. 

Reference- https://docs.microsoft.com/en-us/windows-server/networking/technologies/hcn/hcn-scenarios#scenario-hcn-namespace

Reply:
I know :). You wrote `hcs`

Reply:
Ah right. Missed that 😅

I'll rectify that in the next push.